scripts/exrrfs_fcst.sh

```diff
@@ -28,14 +28,19 @@ else
   ${cpreq} ${COMINrrfs}/${RUN}.${PDY}/${cyc}${ensindexstr}/da/restart.${timestr}.nc .
   do_restart='true'
 fi
-offset=$((10#${cyc}%6))
-CDATElbc=$($NDATE -${offset} ${CDATE})
+
+
+#offset=$((10#${cyc}%6))
+#CDATElbc=$($NDATE -${offset} ${CDATE})
+CDATElbc=${CDATE}
```

Comment:
Line 33-35:
Can we remove line 35 and add a line after line 33:
`if [[  "FCST_ONLY" == "YES" ]]; then offset=0; fi`